test-site/src/pages/LocationsPage.tsx

```diff
@@ -0,0 +1,76 @@
+import { useSearchActions, Matcher, useSearchState, SelectableStaticFilter } from '@yext/search-headless-react';
+import {
+  AppliedFilters,
+  ResultsCount,
+  SearchBar,
+  StandardCard,
+  VerticalResults,
+  LocationBias,
+  MapboxMap,
+  Pagination,
+  MapboxMapProps,
+} from '@yext/search-ui-react';
+import { LngLat, LngLatBounds } from 'mapbox-gl';
+import { useCallback, useLayoutEffect } from 'react';
+import { MapPin } from '../components/MapPin';
+
+export interface Location {
+  yextDisplayCoordinate: {
+    latitude: number,
+    longitude: number
+  } | undefined
+}
+
+const mapboxOptions: MapboxMapProps<Location>['mapboxOptions'] = {
+  zoom: 10
+};
+
+export function LocationsPage() {
+  const searchActions = useSearchActions();
+  const filters = useSearchState(state => state.filters.static);
+  useLayoutEffect(() => {
+    searchActions.setVertical('KM');
+    searchActions.executeVerticalQuery();
+  }, [searchActions]);
+
+  const onDrag: MapboxMapProps<Location>['onDrag'] = useCallback(
```

Comment:
do we want to export the type of this function like we do for `getCoordinate`? then users wouldn't need to reach into the props like this to get the type

Reply:
sure!